distro/configs/superset/dashboard-assets/datasets/PostgreSQL/concepts.yaml

```diff
@@ -0,0 +1,137 @@
+table_name: concepts
+main_dttm_col: null
+description: null
+default_endpoint: null
+offset: 0
+cache_timeout: null
+schema: public
+sql: 'SELECT * FROM concepts '
```

Comment:
Thanks for the PR @enyachoke, do we however need to consume these datasets, or these somewhat place holders/example dataset configs?

Reply:
We don't have to, but if we have any datasets which could conflict, we need to make sure we sync them up.

Reply:
Remove the trailing space in the `SELECT * FROM concepts `
@Ruhanga , yes, those are the default ones that will come with Ozone anyway.